eks-best-practices/kustomization.yaml

```diff
@@ -0,0 +1,13 @@
+resources:
+  - check-amazon-inspector.yaml
+  - check-ami-deprecation-time.yaml
+  - check-cluster-endpoint.yaml
+  - check-cluster-logging.yaml
+  - check-cluster-remote-access.yaml
+  - check-cluster-rolearn.yaml
+  - check-cluster-secrets-encryption.yaml
+  - check-cluster-tags.yaml
+  - check-immutable-tags-ecr.yaml
+  - check-instance-profile-access.yaml
+  - check-public-dns.yaml
+  - check-vpc-flow-logs.
```

Comment:
The kustomization.yaml is used by kustomize. So kubectl -k create will use this and create the output to be put in all.yaml. For initial files there is all.yaml. For later ones, cost-management, eks-best-practices, etc, upto permit-dns, I guess this is not done yet.

Reply:
Added templates/all.yaml for cost-management and eks-best-practices.